src/collective/collectionfilter/interfaces.py

```diff
@@ -172,6 +172,22 @@ class ICollectionFilterSchema(ICollectionFilterBaseSchema):
 class ICollectionSearchSchema(ICollectionFilterBaseSchema):
     """Schema for the search filter."""
 
+    button_text = schema.TextLine(
+        title=_("text_search_button", default=u"Search button text"),
+        description=_("help_header", u"Text of the search button."),
+        required=False,
+        default=_(u"Search"),
+    )
+
+    placeholder = schema.TextLine(
+        title=_(
+            "placeholder_search_input", default=u"Placeholder"
+        ),
+        description=_("help_header", u"Placeholder of the search input."),
+        required=False,
+        default=_(u"Search"),
```

Comment:
Should have no default so the default will be translated based on the language of the site.
Same as the button text